projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/cutsomer-ticketing.ts

```diff
@@ -40,3 +40,20 @@ export function verifyTicketDetailsPageVisit(){
   cy.url().should('match',/http:\/\/.+\/my\-account\/support\-ticket\/[0-9]+/);
   cy.get('cx-customer-ticketing-messages').should('exist');
 }
+
+export function visitTicketDetailsPageForFirstTicket(){
```

Comment:
How about parametrizing this method to accept a ticket position in the table.
Something like:
```suggestion
export function visitTicketDetailsPageForTicketInPosition(ticketPosition: number){
```

Like this, we can reuse the same method when we to access the a ticket in the second, third... n position

Reply:
Done.